Src/AutoFixture/EnumerableExtensions.cs

```diff
@@ -41,5 +48,19 @@ internal static IEnumerable<T> SetItem<T>(this IEnumerable<T> items, int index,
 
             return a.Take(index).Concat(new[] { item }).Concat(a.Skip(index + 1));
         }
+
+        public static object ToTypedArray(this IEnumerable items, Type elementType)
```

Comment:
Perhaps it can be internal?

Reply:
Notice, the `EnumerableExtensions` class is internal, so all the methods are still internal, even though they are marked as `public`. This particular method is indeed internal, I don't want to make it public.